test/unit/rules/no-invalid-link-text-test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-invalid-link-text',
+
+  config: true,
+
+  good: ['<a href="https://myurl.com">Click here to read more about this amazing adventure</a>'],
+
+  bad: [
```

Comment:
We need more examples here for `<LinkTo>` and `{{#link-to}}more{{/link-to}}` and `{{link-to 'more' 'route-name'}}` (confirm the order here, I hate this syntax :sob:).

Reply:
since the linter already prevents inline syntax, I didn't add it.